tests/integration/states/test_vmc_org_users.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
As we can not expose vmware specific data, we are trying to add a testuser.
The underlying invite API checks if the passed user is part of vmware Active Directory or not. If the user is part of it, then only the underlying invite API will add the user to the given org.

As the testuser is some dummy value and is not part of the vmware Active Directory, the invite API just sends 200 OK response though it did not added the user to the org.

so we are testing the flow as:

In present function, add the user (as the ""test@vmware.com" is not part of vmware Active Directory, the underlying API just sends 200 OK response)
so when we call absent function by passing the name as same ""test@vmware.com", it should return "No user found"